repository/Zinc-Seaside.package/ZnZincServerAdaptor.class/instance/responseFrom..st

```diff
@@ -6,7 +6,7 @@ responseFrom: aRequestContext
 	cookies := seasideResponse cookies.
 	fullHeaders := ZnHeaders defaultResponseHeaders.
 	partialHeaders keysAndValuesDo: [ :key :value |
-		fullHeaders at: key add: value greaseString ].
```

Comment:
This is a revert of the changes done in https://github.com/SeasideSt/Seaside/commit/3ebd7d4e685eaf47ed4214b0990b312b6c8d6b2c and I have trouble understanding why git is making this change back. When I try to merge master into this branch, it still shows up as a change...

Reply:
I reverted these changes on the branch using git revert-to-commit.